osgp/platform/osgp-adapter-ws-core/src/main/java/org/opensmartgridplatform/adapter/ws/core/application/services/DeviceManagementService.java

```diff
@@ -14,6 +14,7 @@
 
 import javax.validation.Valid;
 
+import org.apache.commons.lang3.StringEscapeUtils;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.apache.commons.lang3.StringEscapeUtils'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)